src/app-layout/__tests__/drawers.test.tsx

```diff
@@ -169,4 +173,38 @@ describeEachAppLayout(({ size }) => {
     expect(drawerTrigger!.getElement()).toHaveAttribute('aria-controls', 'security');
     expect(wrapper.findActiveDrawer()!.getElement()).toHaveAttribute('id', 'security');
   });
+
+  testIf(size !== 'mobile')('tooltip renders correctly on pointerdown events', () => {
+    const { wrapper } = renderComponent(<AppLayout drawers={[testDrawer]} />);
+    const drawerTrigger = wrapper.findDrawerTriggerById('security')!;
+
+    drawerTrigger!.fireEvent(new MouseEvent('pointerdown', { bubbles: true }));
+    waitFor(() => {
+      expect(wrapper!.findByClassName(tooltipStyles.root)).toBeTruthy();
+      expect(wrapper!.findByClassName(popoverStyles.content)).toBeTruthy();
+    });
+
+    const otherEl = wrapper!.findNavigation();
+    expect(otherEl).toBeTruthy();
+    otherEl.fireEvent(new MouseEvent('pointerdown', { bubbles: true }));
+    waitFor(() => {
+      expect(wrapper!.findByClassName(tooltipStyles.root)).toBeNull();
+    });
+  });
+
+  testIf(size !== 'mobile')('tooltip renders correctly on focus and blue events', () => {
```

Comment:
I will update blue === blur

Reply:
I will also expand to mobile and see if the waitFor is needed or needs to be awaited.